src/pybind/mgr/dashboard/frontend/src/app/shared/components/deletion-modal/deletion-modal.component.html

```diff
@@ -52,7 +49,6 @@
 
 <ng-template #deletionHeading>
   <ng-container i18n>
-    Delete
+    {{ actionDescription | titlecase }} {{ itemDescription | titlecase }}
```

Comment:
Don't use the [titlecase pipe](https://angular.io/api/common/TitleCasePipe) for ``itemDescription``, otherwise ``RBD snapshot`` will be transformed to ``Rbd Snapshot``.